packages/ethereum-contracts/contracts/agreements/InstantDistributionAgreementV1.sol

```diff
@@ -22,6 +22,15 @@ import { AgreementLibrary } from "./AgreementLibrary.sol";
 import { SlotsBitmapLibrary } from "./SlotsBitmapLibrary.sol";
 
 
+/**
+ * @dev The IConstantFlowAgreementV1 implementation
```

Comment:
should be IInstantDistributionAgreementV1 in this file

Reply:
done